.storybook/preview.tsx

```diff
@@ -105,13 +105,18 @@ const adjustedTheme = (ancestorTheme: Theme, theme: Theme) => ({
   ),
 })
 
-export const globalStyles = (theme: Theme) => css`
-  ${normalize()}
+export const globalStyles = (mode: 'light' | 'dark') => (theme: Theme) =>
+  css`
+    ${normalize()}
 
-  body {
-    color: ${theme.colors.neutral.text};
-  }
-`
+    body {
+      color: ${theme.colors.neutral.text};
+    }
+
+    :root {
+      color-scheme: ${mode};
```

Comment:
Should also support system setting 

https://developer.mozilla.org/en-US/docs/Web/CSS/color-scheme#adapting_to_color_schemes

Reply:
@chambo-e we don't have such option on storybook, its or light or dark. Also I don't really see the purpose of supporting it as you change theme on purpose to see component on each individual themes.